Sources/ConsoleKit/Command/Base/Options/Option.swift

```diff
@@ -0,0 +1,66 @@
+/// A type-erased `Option`.
+public protocol AnyOption {
+    
```

Comment:
Vapor style typically omits this line and has the first property touching the type decl.